src/components/SpruceForm/FieldTemplates.tsx

```diff
@@ -6,22 +6,83 @@ import {
   ObjectFieldTemplateProps,
 } from "@rjsf/core";
 import Icon from "components/Icon";
+import { Unpacked } from "types/utils";
 import { SpruceFormContainer } from "./Container";
 import ElementWrapper from "./ElementWrapper";
 
 // Custom field template that does not render fields' titles, as this is handled by LeafyGreen widgets
 export const DefaultFieldTemplate: React.FC<FieldTemplateProps> = ({
   classNames,
   children,
+  description,
+  fields,
   hidden,
-}) => !hidden && <div className={classNames}>{children}</div>;
+  id,
+  label,
+  schema,
+  uiSchema,
+}) => {
+  const isNullType = schema.type === "null";
+  // fields is incorrectly typed in FieldTemplateProps, so cast to avoid type error
+  const { TitleField } = fields as any;
```

Comment:
Is there something else we can cast this too so we don't completely lose typesafety?